bd_tools/__main__.py

```diff
@@ -0,0 +1,58 @@
+"""The betz drive tools package."""
+import argparse
+import sys
+from pathlib import Path
+
+BIN_PATH = Path(__file__).parent / "bin"
+
+
+def get_tools():
+    """Returns all scripts in the bin directory"""
+    tools = []
+    for tool in BIN_PATH.glob("*.py"):
+        tools.append(tool.stem)
+    return tools
+
+
+def parser_args(tools):
+    parser = argparse.ArgumentParser(description="BetzDrive Tools Package")
+    parser.add_argument(
+        "tool", type=str, choices=tools, help=("Name of the tool to use.")
```

Comment:
any reason for the extra pair of parentheses in the help field?

Reply:
I had an append before that crossed to the newline. Can remove.